extensions/sql-database-projects/src/models/project.ts

```diff
@@ -1321,6 +1370,72 @@ export class Project implements ISqlProject {
 		}
 	}
 
+	/**
+	 * Adds a value to project property, separated by semicolon if
+	 * one or more values are already present in project property
+	 * @param propertyName Name of property
+	 * @param valueToAdd Value to be added to project property
+	 */
+	private async addValueToProjectProperty(propertyName: string, valueToAdd: string) {
+		let propertyValue = this.evaluateProjectPropertyValue(propertyName);
+
+		if (propertyValue === undefined) {
+			propertyValue = valueToAdd;
+		} else {
+			if (propertyValue.length > valueToAdd.length
+				&& (propertyValue.startsWith(`${valueToAdd};`)
+					|| propertyValue.endsWith(`;${valueToAdd}`)
+					|| propertyValue.indexOf(`;${valueToAdd};`) > 0)) {
+				return;
+			} else if (propertyValue.length === valueToAdd.length
```

Comment:
Since both blocks are identical (`return;`) the condition can probably be combined.